sql/policy_api.sql

```diff
@@ -128,4 +128,93 @@ CREATE OR REPLACE FUNCTION timescaledb_experimental.show_policies(
     relation REGCLASS)
 RETURNS SETOF JSONB
 AS '@MODULE_PATHNAME@', 'ts_policies_show'
-LANGUAGE C  VOLATILE;
\ No newline at end of file
+LANGUAGE C  VOLATILE;
+
+CREATE OR REPLACE FUNCTION _timescaledb_functions.set_policy_scheduled(hypertable REGCLASS, policy_type TEXT, scheduled BOOL)
+RETURNS INTEGER
+AS $$
+    WITH affected_policies AS (
+        SELECT @extschema@.alter_job(j.id, scheduled => set_policy_scheduled.scheduled)
+        FROM _timescaledb_config.bgw_job j
+            JOIN _timescaledb_catalog.hypertable h ON h.id = j.hypertable_id
+        WHERE j.proc_schema IN ('_timescaledb_internal', '_timescaledb_functions')
+        AND j.proc_name = set_policy_scheduled.policy_type
+        AND j.id >= 1000
+        AND scheduled <> set_policy_scheduled.scheduled
+        AND format('%I.%I', h.schema_name, h.table_name) = set_policy_scheduled.hypertable::text
+    )
+SELECT count(*) FROM affected_policies;
+$$
+LANGUAGE SQL SET search_path TO pg_catalog, pg_temp;
```

Comment:
Nit: instead of `policy_type` it might be better to just pass in the `regproc` ID and then match it with the proc_schema.proc_name cast to a `regproc`. Then it would be impossible to pass in a random text string.


```suggestion
CREATE OR REPLACE FUNCTION _timescaledb_functions.set_policy_scheduled(hypertable REGCLASS, policy_proc REGPROC, scheduled BOOL)
RETURNS INTEGER
AS $$
    WITH affected_policies AS (
        SELECT @extschema@.alter_job(j.id, scheduled => set_policy_scheduled.scheduled)
        FROM _timescaledb_config.bgw_job j
            JOIN _timescaledb_catalog.hypertable h ON h.id = j.hypertable_id
        WHERE format('%I.%I', j.proc_schema, j.proc_name)::regproc = set_policy_scheduled.policy_proc
        AND j.id >= 1000
        AND scheduled <> set_policy_scheduled.scheduled
        AND format('%I.%I', h.schema_name, h.table_name) = set_policy_scheduled.hypertable::text
    )
SELECT count(*) FROM affected_policies;
$$
LANGUAGE SQL SET search_path TO pg_catalog, pg_temp;
```